version/version.go

```diff
@@ -13,5 +13,5 @@
 package version
 
 var (
-	Version = "1.0.0-next"
+	Version = "v0.3.0-next"
```

Comment:
I wonder if we should be fine with just `v0.3.0`, and then in https://github.com/devfile/devworkspace-operator/issues/377 we'll add git commit info.

Reply:
does the `v` prefix break anything in go-speak version semantics?